contrib/docker/docker-compose.prod.yaml

```diff
@@ -34,7 +34,7 @@ services:
       - DEBUG_MODE
       - JITSI_URL
       - JITSI_PRIVATE_MODE
-      - ENABLE_FEATURE_MAP_EDITOR
+      - ENABLE_MAP_EDITOR
```

Comment:
I think we should add `USERS_ALLOWED_TO_MAP_EDITOR` here too (and in the .env.template from /contrib/docker)